### PR TITLE
re-align and change puts to p

### PR DESCRIPTION
**Pull request by klorealf**
solo challenge 4.5

### CODE REVIEW

**Comment by klorealf** [comment]
@Bontgoy Can I please have a code review of this challenge?

File: ruby/vampires.rb [hunk 1]
```diff
@@ -79,24 +79,22 @@
    puts "Allergy: #{loop_allergy_counter+1}"
```
Comment (Bontgoy) [inline ruby/vampires.rb:79]: This section should happen before the final vampire evaluations.  Try to keep your logic organized in a way that the program collects all of the necessary data first and then does a full evaluation.

Collecting the allergy data first and then running returning the vampire results all at once is a better practice vs collecting some of the data, returning an eval and then re-evaluating.

File: ruby/vampires.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-p "How many employees will be processed?"
+  p "How many employees will be processed?"
   number_employees = gets.chomp.to_i
 
 loop_counter = 0
```
Comment (Bontgoy) [inline ruby/vampires.rb:4]: On line 12 you can refactor your name check to a single `if`

`if name == "Drake Cula" || name == "Tu Fang"`

Comment (Bontgoy) [inline ruby/vampires.rb:4]: Line 26

I would like to see you follow the age convention in the instructions.  We want to know if you the employee is lying about their age.

Lets consider checking the current year against how old they say they are and what year they say that they were born in.  Investigate the Ruby `Time` class and see if you can come up with something creative and dynamic.

**Review by Bontgoy — COMMENTED** [review]
@klorealf Good approach and organization here.  I left some refactor notes on how you are finalizing the evaluation as well as a Time class refactor suggestion.

**Pull request merged**